libtextsecure/account_manager.js

```diff
@@ -532,12 +532,38 @@
         'private'
       );
 
+      /*
+      // Ensure that we always have a group conversation for comparisong
+      const debugConversation = await ConversationController.getOrCreateAndWait(
+        'debug',
+        'group'
+      );
+      await debugConversation.setFriendRequestStatus(4);
+      await debugConversation.set(updates);
+      */
+      const updates = {
+        name: 'GroupChat',
+        active_at: Date.now(),
+      };
+
+      // Ensure that we always have a public chat conversation for ourself
+      const publicChatConversation = await ConversationController.getOrCreateAndWait(
+        'lokiPublicChat',
+        'public'
+      );
+      updates.name = 'PublicChat';
+      await publicChatConversation.setFriendRequestStatus(4);
```

Comment:
`window.friends.friendRequestStatusEnum.friends` or you could add `friends` to the global comment at top of file and leave out `window`

Reply:
the global comment, is that a grunt thing?